tests/memoryview/numpy_memoryview.pyx

```diff
@@ -32,18 +32,16 @@ def ae(*args):
         if x != args[0]:
             raise AssertionError(args)
 
-__test__ = {}
-
-def testcase(f):
-    __test__[f.__name__] = f.__doc__
-    return f
-
 def testcase_numpy_1_5(f):
     major, minor, *rest = np.__version__.split('.')
-    if (int(major), int(minor)) >= (1, 5):
-        __test__[f.__name__] = f.__doc__
+    if not (int(major), int(minor)) >= (1, 5):
+        f.__doc__ = ""  # disable the tests
```

Comment:
It blows my mind that something _older_ than numpy 1.5 (from [2010](https://github.com/numpy/numpy/tags?after=v1.5.1rc1)!) still has test paths in Cython. Why not rip those out? Can these positively primeval numpy versions still be considered supported in any realistic way...?

Reply:
I've removed that them. I hadn't paid much attention to the actual version numbers here, but it looks fairly pointless to me. I can always undo it if there's a good reason to test old Numpy versions that I've missed.